executor/android/android_seccomp.h

```diff
@@ -0,0 +1,318 @@
+/*
+ * Copyright (C) 2017 The Android Open Source Project
```

Comment:
Is this whole file copied from Android?

 @dvyukov
dvyukov 28 days ago Member
It mentioned GOARCH_amd64, so I assume it did not come from Android?

 @dvyukov
dvyukov 28 days ago Member
But even if it come from Android, this is a forked copy, right?

Reply:
Well it was copied then modified, does that mean the copyright goes away?  I assume not right?

Reply:
I dunno about copyright message, I guess it needs to stay if the code was copied then modified. But maybe we need to have 2 copyrights now as now this is owner by syzkaller project effectively and will be modified by "syzkaller authors" and they will hold copyright.

Reply:
I am more interested in how much control we have over this file. Is it copied verbatim from somewhere else and will be overwritten on the next update, or it's our fork that we can change.
This is our fork, right?

Reply:
Well, it can change in Android R so we would want to sync it over if we wanted to make sure we continue to do the right thing.